products/digital-sales-rooms/saas/index.md

```diff
@@ -0,0 +1,16 @@
+---
+nav:
+  title: SaaS
+  position: 50
+
+---
+
+# Digital Sales Rooms with SaaS
+If you are Beyond merchant and using SaaS, the Digital Sales Rooms plugin was installed in your SaaS instance. So, you should see the Digital Sales Rooms section in Marketing menu item.
+However, there are some stuffs need to complete the whole project:
```

Comment:
what does complete whole project mean?

Reply:
it means making the DSR works with SaaS completely.